src/Tests/README.md

```diff
@@ -0,0 +1,111 @@
+# Tests
+
+ONE-vscode's unittests are based on [mocha](https://mochajs.org/) and [@vscode/test-electron](https://github.com/microsoft/vscode-docs/blob/main/api/working-with-extensions/testing-extension.md).
+- use @vscode/test-electron because ONE-vscode uses vscode api
+  - use Extension Development Host which supports full access to the VS Code API
+- interface of mocha: TDD (BDD to test UX will be introduced in the future)
+- use `assert` of [chai](https://www.chaijs.com/api/assert/)
+
+## structure
+
+Tests are placed in `src/Tests`. The structure follows `src/`.
+```
+src/
+  ...
+  Project/
+    Builder.ts
+    Job.ts
+    ...
+  Utils/
+    Helpers.ts
+    ...
+```
+```
+src/
+  ...
+  Project/
+    Builder.ts
+    Job.ts
+    ...
+  Utils/
+    Helpers.ts
+    ...
+  Tests/
+    ...
+    Project/
+      Builder.test.ts
+      Job.test.ts
+      ...
+    Utils/
+      Helpers.test.ts
+      ...
+```
+
+## scripts
+
+There are three scripts for tests.
+- runTest.ts: the test script
+  - uses the `@vscode/test-electron` API to simplify the process of downloading, unzipping, and launching VS Code with extension test parameters
+  - `--extensionTestsPath` points to the test runner script
+- index.ts: the test runner script
+  - programmatically runs the test suite
+  - exactly does glob() to find tests after adding `hooks` and then, runs them
+- hooks.ts: global hooks
+
+## How to run/debug tests?
+
+```
+ONE-vscode$ npm test
+```
+
+OR
+
+Click F5 or run the `Extension Tests` with breaking points for debug.
```

Comment:
I guess you meant 
`Press F5 and then run the Extension Tests`.
Is `Extension Tests` a menu name? Could you please help us to find where `Extension Tests` can be found?

Reply:
It means `Extension Tests` configured in launch.json.
![image](https://user-images.githubusercontent.com/10216715/158563953-9de3c2c3-d00a-43cf-9990-d3bf98cef107.png)

Reply:
How about the something like the following?
```
Select `Extension Tests` in debug configuration and then run. (e.g., by pressing F5)
```

What I found was, in my case just pressing F5 does not run `Extention Tests`. F5 runs `Launch Extention` in my environment. Therefore, before pressing F5, I had to select `Extention Tests` in my debug configuration.

Reply:
👍

Reply:
```suggestion
Select `Extension Tests` in debug configuration and then run. (e.g., by pressing F5 and with breaking points for debug if you need to debug tests.)
```